dom.bs

```diff
@@ -6132,10 +6182,8 @@ interface DOMImplementation {
   Returns a <a>doctype</a>, with the given
   <var>qualifiedName</var>, <var>publicId</var>, and
   <var>systemId</var>. If <var>qualifiedName</var> does not
-  match the <code><a type>Name</a></code> production, an
-  "{{InvalidCharacterError!!exception}}" {{DOMException}} is thrown, and if it does not match the
-  <code><a type>QName</a></code> production, a
-  "{{NamespaceError!!exception}}" {{DOMException}} is thrown.
+  match the <code><a>QName</a></code> production, an
+  "{{InvalidCharacterError!!exception}}" {{DOMException}} is thrown.
```

Comment:
What if instead this called the new "validate and extract" algorithm?

Reply:
In chromium the code for parsing this is shared between this and the attributeNS methods, so I'd have to split them in order to make this one keep using QName.

Reply:
Actually this probably depends on the discussion here: https://github.com/whatwg/dom/issues/1373

Reply:
Yep, this has been re-done.